extensions/narayana-lra/runtime/src/main/java/io/narayana/lra/client/internal/proxy/nonjaxrs/LRAParticipant.java

```diff
@@ -1,24 +1,3 @@
-/*
- * JBoss, Home of Professional Open Source.
- * Copyright 2019, Red Hat, Inc., and individual contributors
- * as indicated by the @author tags. See the copyright.txt file in the
- * distribution for a full listing of individual contributors.
- *
- * This is free software; you can redistribute it and/or modify it
- * under the terms of the GNU Lesser General Public License as
- * published by the Free Software Foundation; either version 2.1 of
- * the License, or (at your option) any later version.
- *
- * This software is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this software; if not, write to the Free
- * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
- * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
- */
```

Comment:
So... these files are a problem for me:
- first, Quarkus is Apache licensed so we need to relicense them under the right license. I have no idea where they come from so I'll let you figure out if it's doable or not;
- second are they somehow overriding files in existing jars? It looks like it from what I can see in the other jars. That's definitely something we don't want to do: we don't want a split package and we don't want to override classes this way. I have no idea why you did that so we would need to understand the reason to help you solve this problem.

Reply:
These files were copied over from the narayana implementation of LRA, I needed access to the constructor which is package private. I have changed them in the narayana repo (https://github.com/jbosstm/narayana/pull/1888) so when we next do a narayana release I will update this extension and remove the two files (LRAParticipant.java and LRAParticipantRegistry.java).